events/syscall/launch_package_management_process_in_container.go

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(
+	LaunchPackageManagementProcessInContainer,
+	events.WithDisabled(), // this rules is not included in falco_rules.yaml (stable rules), so disable the action
+)
+
+func LaunchPackageManagementProcessInContainer(h events.Helper) error {
+	// Make sure it runs in container and user.name != _apt
+	if h.InContainer() {
+		if os.Getenv("USER") == "_apt" {
```

Comment:
Can you expand on the need for this change?

Reply:
in the [rule](https://github.com/falcosecurity/rules/blob/fbf0a4e8ce6b56a9c5ec4e6c4abadfaf14c9786b/rules/falco-incubating_rules.yaml#L944) conditions user.name != _apt.
Due to that I am ensuring in the code that user.name != _apt.

Reply:
Uh nice! Didn't know that :)